src/helpers/mixed.js

```diff
@@ -102,7 +102,7 @@ export function _injectProductInfo(key, element) {
       if (!warningMessage) {
         if (releaseTime > keyGenTime + 1) {
           warningMessage = toSingleLine`
-          Your license key of Handsontable Pro has expired.‌‌‌‌ 
+          Your license key of Handsontable Pro has expired.‌‌‌‌
```

Comment:
Here you ate space character that is why after built I've got "...has expired.‌‌‌‌Renew your mainte..." in the console.